paseos/actors/actor_builder.py

```diff
@@ -165,6 +166,78 @@ def set_power_devices(
             + f"ChargingRate={charging_rate_in_W}W to actor {actor}"
         )
 
+    def set_thermal_model(
+        actor: SpacecraftActor,
+        actor_mass: float,
```

Comment:
At the moment we don't need it anywhere else. I would avoid adding a function to set mass independently until it is actually used elsewhere. Otherwise, it can both mislead users that it is used elsewhere and secondly adding a thermal model then requires multiple function calls (unlike other physical models)